docs/tips.md

```diff
@@ -445,3 +445,34 @@ While Yazi is already fast, there is still plenty of room for optimization for s
 - For users managing network files, it's recommended to disable all previewers and preloaders since previewing and preloading these files means they need to be downloaded locally.
 - For low-spec devices like Raspberry Pi, [reducing the concurrency](/docs/configuration/yazi#tasks) will make Yazi faster since the default configuration is optimized for PCs, and high concurrency on these low-spec devices may have the opposite effect.
 - For users who don't need accurate mime-type, [`mime.yazi`](https://github.com/DreamMaoMao/mime.yazi) may be useful, as it simply returns mime-type based on file extensions, while Yazi defaults to obtaining mime-type based on file content for accuracy. Mime-type is used for matching opening, previewing, rendering rules. Encourage users to choose the appropriate `mime` plugin based on their needs, which is why we decided to open it up to plugin developers.
+
+## Use Yazi as a file tree picker in Helix with Zellij
+
+Yazi can be used as a file picker to browse and open a file in your current Helix instance (running in a Zellij session). 
+
+Add a keymap to your Helix config, e.g.:
+
+```toml
+# config.toml
+[keys.normal."'"]
+y = ":sh zellij run --floating -n 'yazi picker' -- /home/rboynton/.config/fish/scripts/open_in_helix_from_yazi.fish"
```

Comment:
```suggestion
y = ":sh zellij run --floating -n 'yazi picker' -- fish /home/rboynton/.config/fish/scripts/open_in_helix_from_yazi.fish"
```

A bit picky, but just in case someone forgets to make the file executable 😁. The error given by Zellij in this case is very weird.

Reply:
@postsolar thank you for being my tester 😆. What was the error?

Reply:
If I try to run a non-executable file with Zellij, the output is
> fatal runtime error: assertion failed: output.write(&bytes).is_ok()